app/src/test/java/mobile/programming/musicapp/InstrumentDataIntegrationTest.kt

```diff
@@ -32,14 +32,12 @@ class InstrumentDataIntegrationTest {
     }
 
     private fun checkIfInstrumentDataIsEmpty() {
-        var instrumentlist = ArrayList<Instrument>()
+        var instrumentList = ArrayList<Instrument>()
         mvm.instruments.observeForever {
-
-            instrumentlist = it
+            instrumentList = it
         }
-
         Thread.sleep(5000)
-        assertNotNull(instrumentlist)
-        assertTrue(instrumentlist.size > 0)
+        assertNotNull(instrumentList)
+        assertTrue(instrumentList.size > 0)
```

Comment:
Noticed that this variable could use some camel casing, as recommended in https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules